net/net.sh

```diff
@@ -512,8 +269,8 @@ startBootstrapLeader() {
          \"$internalNodesStakeLamports\" \
          \"$internalNodesLamports\" \
          $nodeIndex \
-         $numBenchTpsClients \"$benchTpsExtraArgs\" \
-         $numBenchExchangeClients \"$benchExchangeExtraArgs\" \
+         ${#clientIpList[@]} \"$benchTpsExtraArgs\" \
+         ${#clientIpList[@]} \"$benchExchangeExtraArgs\" \
          \"$genesisOptions\" \
```

Comment:
@mvines to asynchonrously configure for various clients in genesis and then start any number of them later requires this change here, which just ends up adding potentially unused client-accounts.yml files passed as `--primordial-accounts-file` to genesis.  I don't think this should cause any issue, wdyt?

Reply:
Should be fine.  We no longer use `net/` for production clusters so that makes it easier to do things like this

Reply:
Yeah, that was my thinking as well, thanks.